docs/docusaurus/sidebars.json

```diff
@@ -65,7 +77,12 @@
         "ids": ["lte/tr069", "lte/s1ap_tests", "lte/dev_notes"]
       }
     ],
-    "Feature Guides": ["howtos/ue_metering", "howtos/config_agw_bridged", "howtos/thanos"],
+    "Feature Guides": [
+      "howtos/ue_metering",
+      "howtos/config_agw_bridged",
+      "howtos/thanos",
+      "howtos/call_tracing"
```

Comment:
I'm trying to understand the right place we should add this guide. I feel the right fit is NMS, but will probably broadcast it in the team and get some broader consensus here.

Reply:
Alternatively, if you can add the ### Initiating a Call Trace and ### Viewing a call trace to NMS user guide also that will be great. Thanks.

Reply:
I think I'm gonna make a separate PR for these larger changes to re-organize the docs